### ISSUE TITLE
Channel selector

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bigman73/practicesharp#1]
```
Great software, but it needs still one thing.

I'd like to use it for studying aebersold play-a-long but...

...I can't switch the channels in double mono recs!

it would be great a selector, but greater a slide that merges the volumes of 
the two channels in the way you like!


Compliments for the project. Look for .net framework 4


```

Original issue reported on code.google.com by `gegg...@gmail.com` on 2 Mar 2011 at 11:44

**Comment by bigman73** [comment]
Started to work on this feature. Will work in an upcoming Practice version (2.0.1)

**Comment by bigman73** [comment]
Implemented as of version 2.0.5
![image](https://user-images.githubusercontent.com/5384551/34966400-585f4d84-fa29-11e7-88b5-6e587a78dfeb.png)
